creating-an-installation-medium-for-ubuntu-22-04: add ISO image note (#3847)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-installation-medium-for-ubuntu-22-04.adoc

# rm -f {iso_image}
----

> As an alternative to copying the content of the ISO image, one may instead mount the ISO image somewhere, such as under `/mnt`,

Reviewer: ```suggestion
. As an alternative to copying the ISO image content, you can mount the image to a directory such as `/mnt`,
```

> then create `/var/www/html/pub/installation_media/ubuntu/22.04-x86_64` as a symbolic link to the mounted image. The ISO image

Reviewer: ```suggestion
. Create a symbolic link at `/var/www/html/pub/installation_media/ubuntu/22.04-x86_64` that points to the mounted image.
```

> should not be mounted directly under `/var/lib/foreman`, because on Debian and Ubuntu this will cause the package postinst script

Reviewer: ```suggestion
Do not mount the ISO image directly under `/var/lib/foreman`.
```

> to fail at the `chmod -Rf /var/lib/foreman` command.

Reviewer: ```suggestion
On Debian and Ubuntu, mounting under this directory causes the package `postinst` script to fail during the `chmod -Rf /var/lib/foreman` command.
```

> On Debian and Ubuntu, mounting under this directory causes the package `postinst` script to fail during the `chmod -Rf /var/lib/foreman` command.

Author: Applied your suggestion, and added a sentence about the ISO remaining mounted, since this is an alternative to copying the image content into the webserver's docroot.

# rm -f {iso_image}
----

> +

Reviewer 2: ```suggestion
```
We can drop this because the NOTE is placed after the whole procedure.

[NOTE]
====
As an alternative to copying the ISO image content, you can create a symbolic link at `/var/www/html/pub/installation_media/ubuntu/22.04-x86_64` that points to the mounted image.

> The ISO image will then remain mounted so that it is accessible to HTTP clients.

Reviewer 2: Suggestion for wording with active voice to make it clearer that it's an action for the user.
```suggestion
Keep the ISO image mounted so that it is accessible to HTTP clients.
```
